.devcontainer/advanced-integration-v1/devcontainer.json

```diff
@@ -6,15 +6,19 @@
   // Use 'onCreateCommand' to run commands when creating the container.
   "onCreateCommand": "bash ../../.devcontainer/advanced-integration-v1/welcome-message.sh",
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": "npm install",
+  "postCreateCommand": {
+    "Install Backend Packages": "cd backend/node && npm install",
+    "Install Frontend Packages": "cd frontend/html && npm install"
+  },
   // Use 'postAttachCommand' to run commands when attaching to the container.
   "postAttachCommand": {
-    "Start server": "npm start"
+    "Start server": "cd backend/node && npm start",
```

Comment:
Why do we need backend and frontend folders . Can we just have client and server folders with respective language folders similar to this repo : https://github.com/paypal-examples/fastlane-sample-application/tree/main/server

Reply:
I can rename them to sever and client